test/Dashboard.EndToEndTests/Tests/ArgumentsDisplay/QueueArgumentsDisplayFunctions.cs

```diff
@@ -23,13 +23,13 @@ public class QueueArgumentsDisplayFunctions
         public static readonly MethodInfo SingletonMethodInfo = typeof(QueueArgumentsDisplayFunctions).GetMethod("Singleton");
 
         [Singleton("TestScope")]
-        public static void Singleton(
+        public static async Task Singleton(
             [QueueTrigger("queue-singleton-in")] string input,
-            [Queue("queue-singleton-out")] out string output,
-            [Queue(DoneNotificationFunction.DoneQueueName)] out string done)
+            [Queue("queue-singleton-out")] IAsyncCollector<string> output,
```

Comment:
All the below changes are changing the nature of the tests. E.g. this one here is explicitly testing a singleton out and the queue naming matches that. Hopefully we can just leave these tests as is - we actually want test coverage for those things.